broker/src/main/java/io/camunda/zeebe/broker/bootstrap/DiskSpaceUsageMonitorStep.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.bootstrap;
+
+import io.camunda.zeebe.broker.system.monitoring.DiskSpaceUsageMonitor;
+import io.camunda.zeebe.util.FileUtil;
+import io.camunda.zeebe.util.sched.ConcurrencyControl;
+import io.camunda.zeebe.util.sched.future.ActorFuture;
+import java.io.File;
+import java.io.IOException;
+
+public class DiskSpaceUsageMonitorStep extends AbstractBrokerStartupStep {
+
+  @Override
+  void startupInternal(
+      final BrokerStartupContext brokerStartupContext,
+      final ConcurrencyControl concurrencyControl,
+      final ActorFuture<BrokerStartupContext> startupFuture) {
+
+    final var data = brokerStartupContext.getBrokerConfiguration().getData();
+    try {
+      FileUtil.ensureDirectoryExists(new File(data.getDirectory()).toPath());
```

Comment:
:wrench: Note that this will break compilation since you now need to import `Paths`, so it cannot be auto committed.

```suggestion
      FileUtil.ensureDirectoryExists(Paths.get(data.getDirectory()));
```